### ISSUE TITLE
Can't retrieve docs for defintions

### ISSUE THREAD

**Issue by KevinSjoberg** [mauricioszabo/atom-chlorine#230]

**Comment by KevinSjoberg** [comment]
I've confirmed that saving the file to disk causes documentation lookup to start working as expected.

Would it be possible to support unsaved files as well?

**Comment by mauricioszabo** [comment]
Yes, I'll check what's wrong :). I'll keep the bug open here to track this issue
